engine/src/main/java/org/terasology/entitySystem/sectors/SectorSimulationEvent.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.entitySystem.sectors;
+
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.Event;
+import org.terasology.module.sandbox.API;
+
+/**
+ * This is the event sent to all sector-level entities by the {@link SectorSimulationSystem}, allowing them to do simulation. It
+ * is sent regardless of whether the chunk the entity is in is loaded or not.
+ *
+ * @see SectorSimulationEvent#getDelta()
+ */
+@API
+public class SectorSimulationEvent implements Event {
+    private float delta;
+
+    /**
+     * @see SectorSimulationEvent#getDelta() for the delta parameter
+     */
+    protected SectorSimulationEvent(float delta) {
+        this.delta = delta;
+    }
+
+    /**
+     * This gives the time elapsed, in seconds, since the last time this event was sent to the given {@link EntityRef}.
```

Comment:
I'm wondering whether it is a smart decision to use _seconds_ here. It obviously conveys the idea that sector simulation happens at larger intervals. However, usually the simulations would get a delta in _milliseconds_ for their computation. This might yield confusion when you try to use the same simulation code for both scopes.